migrations/update_activity_log_table.php.stub

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class UpdateActivityLogTable extends Migration
```

Comment:
Please rename to `AddEventColumnToActivityLogTable` - otherwise we will have problems with the next migration.